docs/docs/installation/configuring-superset.mdx

```diff
@@ -69,6 +69,32 @@ you can add the endpoints to `WTF_CSRF_EXEMPT_LIST`:
 WTF_CSRF_EXEMPT_LIST = [‘’]
 ```
 
+### Using a production metadata database
+
+By default Superset is configured to use SQLite, it's a simple and fast way to get you started
+(no installation needed) but for production environments you should use a different database engine on
+a separate host or container.
+
+Superset supports the following database engines/versions:
+
+| Database Engine                                           | Supported Versions                |
+| --------------------------------------------------------- | --------------------------------- |
+| [MySQL](https://www.mysql.com/)                           | 5.X                               |
+| [PostgreSQL](https://www.postgresql.org/)                 | 10.X, 11.X, 12.X, 13.X, 14.X      |
```

Comment:
Could we kinda recommend using Postgres by placing it on top in the list? Or maybe even explicitly recommend it?
```suggestion
Superset supports the following database engines/versions:

| Database Engine                                           | Supported Versions                |
| --------------------------------------------------------- | --------------------------------- |
| [PostgreSQL](https://www.postgresql.org/)                 | 10.X, 11.X, 12.X, 13.X, 14.X      |
| [MySQL](https://www.mysql.com/)                           | 5.X                               |
```